psi4/driver/procrouting/proc.py

```diff
@@ -4352,7 +4352,8 @@ def run_dlpnomp2(name, **kwargs):
     """
     optstash = p4util.OptionsState(
         ['DF_BASIS_MP2'],
-        ['SCF_TYPE'])
+        ['SCF_TYPE'],
+        ['DLPNO_ALGORITHM'])
```

Comment:
```suggestion
        ["DLPNO", "DLPNO_ALGORITHM"])
```
also 2 places below. unless the option is in the globals section of read_options, it needs its module listed here.